source/install/install-ubuntu-2004.rst

```diff
@@ -0,0 +1,23 @@
+.. _install-ubuntu-2004:
+
+=========================================
+Installing Mattermost on Ubuntu 20.04 LTS
+=========================================
+
+Install a production-ready Mattermost system on up to three machines.
+
+.. include:: install-common-intro.rst
+
+.. contents:: Install and configure the components in the following order. Note that you need only one database, either MySQL or PostgreSQL.
+  :backlinks: top
+  :local:
+
+.. include:: install-ubuntu-2004-server.rst
+.. include:: install-ubuntu-2004-mysql.rst
+.. include:: install-ubuntu-2004-postgresql.rst
+.. include:: install-ubuntu-1804-mattermost.rst
```

Comment:
Is the Ubuntu-1804-mattermost guide the same as for Ubuntu 20.04?

Reply:
@jasonblais I checked two different articles before starting so yeah they are quite similar